common/eth2_network_config/built_in_network_configs/mainnet/config.yaml

```diff
@@ -1,14 +1,14 @@
 # Mainnet config
 
 # Extends the mainnet preset
-PRESET_BASE: 'mainnet'
+PRESET_BASE: "mainnet"
```

Comment:
We should maintain same format as upstream file https://github.com/ethereum/consensus-specs/blob/dev/configs/mainnet.yaml